fe/fe-core/src/main/java/com/starrocks/planner/OlapScanNode.java

```diff
@@ -412,17 +414,21 @@ public void addScanRangeLocations(Partition partition,
             internalRange.setTablet_id(tabletId);
 
             // random shuffle List && only collect one copy
-            LocalTablet localTablet = (LocalTablet) tablet;
             List<Replica> allQueryableReplicas = Lists.newArrayList();
             List<Replica> localReplicas = Lists.newArrayList();
-            localTablet.getQueryableReplicas(allQueryableReplicas, localReplicas,
+            tablet.getQueryableReplicas(allQueryableReplicas, localReplicas,
```

Comment:
computeRangeLocations() also call getQueryableReplicas, maybe you should also fix it

Reply:
Fixed